Sources/Signal.swift

```diff
@@ -1506,6 +1506,25 @@ extension Signal {
 	public func debounce(_ interval: TimeInterval, on scheduler: DateScheduler) -> Signal<Value, Error> {
 		return flatMapEvent(Signal.Event.debounce(interval, on: scheduler))
 	}
+
+	/// Forward the latest values on `scheduler` every `interval`.
+	///
+	/// - seealso: `throttle`
+	/// - seealso: `debounce`
+	///
+	/// - note: If the input signal terminates, the returned signal will
+	///         terminate immediately without forwarding the values
+	///	        currently being accumulated.
+	///
+	/// - parameters:
+	///   - interval: A repetition interval.
+	///   - scheduler: A scheduler to send values on.
+	///
+	/// - returns: A signal that sends all values that are sent from `self` at
+	///            `interval` seconds apart.
+	public func chunk(_ interval: DispatchTimeInterval, on scheduler: DateScheduler, ignoreEmptyChunks: Bool = false) -> Signal<[Value], Error> {
```

Comment:
I'm very hesitant to introduce complete new operator names. I think our grouping of operators by base name has been very successful and would like for us to continue it.

While this is similar to `debounce`/`throttle`, I'm not sure it's really throttling them. I don't think a throttle is always regular.

I think this could fit under `collect` though.

```swift
func collect(every interval: DispatchTimeInterval, on scheduler: DataScheduler) -> Signal<[Value], Error> {
```

`collect` serves as the base name for operators that go from `A` to `[A]`.

Thoughts @ReactiveCocoa/reactiveswift?

Reply:
Oh yep I forgot to say that bikeshedding the name is very welcome, I have no attachments to the one I chose.